src/flb_config_map.c

```diff
@@ -650,11 +650,11 @@ int flb_config_map_set(struct mk_list *properties, struct mk_list *map, void *co
             *m_s_num = m->value.val.s_num;
         }
         else if (m->type == FLB_CONFIG_MAP_TIME) {
-            m_i_num = (int *) (base + m->offset);
-            *m_i_num = m->value.val.s_num;
+            m_s_num = (size_t *) (base + m->offset);
```

Comment:
You can't arbitrarily change this because this affects how those settings declared as `FLB_CONFIG_MAP_TIME` such as [this one ](https://github.com/fluent/fluent-bit/blob/master/src/flb_upstream.c#L69) whoses storage is defined [here](https://github.com/fluent/fluent-bit/blob/master/include/fluent-bit/flb_network.h#L46) are processed and you are effectively overwriting adjacent memory (regardless of structure alignment).

Reply:
@leonardo-albertovich, I made this change for the cases in `flb-it-config_map`.

- In the test case of `flb-it-config_map`,  FLB_CONFIG_MAP_TIME is related with `time_t` variable inside the plugin context.
https://github.com/fluent/fluent-bit/blob/master/tests/internal/config_map.c#L16
https://github.com/fluent/fluent-bit/blob/master/tests/internal/config_map.c#L71-L77
In this case I have to update the i_num to s_num since time_t is 8 bytes.

- On the other hand, the link you shared shows FLB_CONFIG_MAP_TIME is related with int variable inside the plugin context, 
- So we need keep things consistent, either using int for FLB_CONFIG_MAP_TIME everywhere or use time_t.

please let me know which do you prefer, thanks!

Reply:
Precisely, the test case seems to be wrong so instead of changing this the test case needs to be changed to conform with the rest of the system.

Reply:
Please revert this.